codespell_lib/data/dictionary.txt

```diff
@@ -12616,7 +12633,7 @@ syncting->syncing
 syndonic->syntonic
 synonomous->synonymous
 synonymns->synonyms
-synopsys->synopsis
+synopsys->synopsis, disabled due to being a company name
```

Comment:
We have to balance false positives with false negatives. My sense is that we are better off leaving this in, and allowing the subset of people/codebases that use this name to whitelist it themselves

Reply:
Ok, I understand. I'll reenable 'synopsys->synopsis'.

Thank you for the review!